modules/extensions/src/views/lite/components/Dropdown.jsx

```diff
@@ -1,21 +1,28 @@
 import React from 'react'
 import Select from 'react-select'
 import Creatable from 'react-select/lib/Creatable'
+import _ from 'lodash'
+import { renderUnsafeHTML } from '../utils'
 
 export class Dropdown extends React.Component {
   state = {
-    options: []
+    options: [],
+    escapeHTML: false
   }
 
-  componentDidMount() {
+  async componentDidMount() {
+    const { data } = await this.props.bp.axios.get('/mod/extensions/config')
+    if (data) {
+      const escapeHTML = _.get(data, 'security.escapeHTML', false)
+      this.setState({ escapeHTML })
+    }
```

Comment:
The extensions module is secure by default, which means that this call will fail for users not authenticated on the studio. 

Also, you don't even need to make this call. As a custom component, you have access to channel-web's store, which already has the configuration loaded.

Simply use ` this.props.store.escapeHTML`

Reply:
yes this makes sense, i had my doubts as this settings should be centralized @spg